GOALS.md

```diff
@@ -0,0 +1,30 @@
+# Why Goals
+Stating goals (and non-goals) are intended to sharpen the join understanidng the the kube-aws product vision. Expressiong the vision by 'Goals' seem to be understandable approach.
+
+#Goals
+Kube-aws is Command Line Tool (CLI) that allow me as experienced console-user to:
+
+* to plan and prepare my kubernetes cluster alongside need AWS resources
+* browse through intermediate details of the configuration (of not yet provisioned clusters)
+
+Kube-aws CLI's aims to support different experience levels of the users in terms of 
+* kubernetes administration expertise
+* CoreOS administration expertise
+* Amazone Web Services expertise
```

Comment:
I've talked with @pieterlange and now I'm a bit afraid to state this kind of things in our documentation.
The point is that, we don't know how our users vary in their levels of expertise!

An extreme case but targeting e.g. users don't know AWS at all, in this stage of kube-aws, would make us all unhappy - possibly endless user support and no time to develop features.

So, in the meantime, how about we rewrite goals targeting users with a certain level of expertise?
Maybe after certain period of time with various fixes, improvements, features which would be written in our roadmap introduced to kube-aws, we can gradually re-target our users?

Btw, I don't mean to make kube-aws targeted to advanced users more than today.
It is already advanced henec our current users are certainly advanced 😉 
I just want to make it extra sure that we don't target to "wrong" users at this stage.